### ISSUE TITLE
PACT: Final QA / UAT for launch

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#18778]
## Definition
We need to QA PACT scenarios end to end for launch, as well as content.

There are high-level scenarios (listed below) with various 'sub-scenarios', and they all must be tested.

- 1990 or earlier
- 1989 or earlier
- During both of these time periods

Note: Fran has already started a QA checklist.

## ACs

- [ ] Create QA flow/checklist
- [ ] All logic flows pass
- [ ] All content passes
- [ ] Design review
- [ ] a11y Review

Note to Fran: If a checklist isn't used, then list the scenarios above.

**Comment by jilladams** [comment]
Talked about this in scrum. 

Flagged: The Mural uses numbering for questions, and those numbers are no longer tied to the copy for the questions. Team suggestion was: let's update the Mural to include the first few words of each question copy, as a cross-reference. With that in place, we'll revisit with the team and it might be sufficient for our purposes to allow the team to verify the logic using Mural vs. a spreadsheet. 

Also: yesterday in Slack, VA verified that CAIA will verify the copy, so we can stick to the logic in our test plan (woo!).

@FranECross , meantime could you add a link to your test plan spreadsheet, just so we have it if we decide to go that route?

**Comment by jilladams** [comment]
From refinement convo: 
Each PACT URL is structured to match the question. e.g. Burn Pits question 2.1 will have the URL https://staging.va.gov/pact-act-eligibility/burn-pit-2-1. 

So: we can use the URL structure as the way to verify the logic flow and make sure testers know they have landed in the right place. (Rather than including the first few words of copy for each question in the flow mural, since copy is evolving). 

There are some questions with A and B variants. That's not currently reflected in the flow mural. @thejordanwood is gonna help and add those. 

At that point, we'll triple check if the mural is sufficient for guiding a tester through verifying the logic.

**Comment by thejordanwood** [comment]
@jilladams To clarify, all A and B variants were shown in the flow but they didn't have a question number above them that corresponds to a URL. I've now added that number above them to make this clear!

**Comment by jilladams** [comment]
Yes, good clarification and I updated my statement to be more clear. 

I just verified the Agent Orange flows pretty easily, with this update. I think we should be set to do the testing with this, but we do need to discuss how to record issues / bugs where we don't get what we expected. 

@laflannery @thejordanwood I know you all used a Mural when you tested header/footer. If we used this logic flow and had testers add a sticker / emoji for broken steps of the flow (if any), does that seem appropriate to y'all? Once QA has signed off / we're clear to ship, we could clean out the QA stickies /notes.

**Comment by jilladams** [comment]
Verified in scrum: Using Mural is good. ACs updated to reflect that.
